Assets/Tests/InputSystem/CoreTests_Actions_Interactions.cs

```diff
@@ -240,30 +241,49 @@ public void Actions_CanPerformHoldInteraction()
         var action = new InputAction(binding: "<Gamepad>/{primaryAction}", interactions: "hold(duration=0.4)");
         action.Enable();
 
+        TestHoldInteraction(action, gamepad.buttonSouth);
+    }
+
+    [Test]
+    [Category("Actions")]
+    public void Actions_CanPerformHoldInteraction_WithMultipleBindings()
+    {
+        //var gamepad = InputSystem.AddDevice<Gamepad>();
+        var keyboard = InputSystem.AddDevice<Keyboard>();
+
+        var action = new InputAction(binding: "<Keyboard>/space", interactions: "hold(duration=0.4)");
+        action.AddBinding("<Keyboard>/s");
+        action.Enable();
+
+        TestHoldInteraction(action, keyboard.spaceKey);
+    }
+
+    void TestHoldInteraction(InputAction action, ButtonControl button)
```

Comment:
Please no methods like this. Makes it hard to understand the tests. Would actually recommend simply folding `_WithMultipleBindings` into the other test.